src/control-plane/backend/lambda/api/common/quicksight-visual-utils.ts

```diff
@@ -0,0 +1,191 @@
+/**
```

Comment:
@tyyzqmf should we move this file and all quicksight templates to `store` or `service`?

Reply:
It is best to move the file to `service`, and add some quicksight SDK action in `store`.

Reply:
fixed